code/modules/projectiles/gun_system.dm

```diff
@@ -1619,8 +1637,12 @@
 
 	if(((flags_item & WIELDED) && wielded_stable()) || CHECK_BITFIELD(flags_item, IS_DEPLOYED) || (master_gun && CHECK_BITFIELD(master_gun.flags_item, WIELDED) && master_gun.wielded_stable()))
 		gun_accuracy_mult = accuracy_mult
+		scatter = max(max(min_scatter, 0), min((scatter + scatter_increase) - ((world.time - last_fired - 1) * scatter_decay), max_scatter))
```

Comment:
what's the -1 for? Doesn't that mean decay will increase your scatter if you've fired in the last second?

Reply:
thats game ticks, it is so it counts 0 on next tick and doenst care about it, think it should be according to firerate though or just not at all and care all ticks